src/monitord/monitor_agents.c

```diff
@@ -20,7 +20,7 @@ void monitor_agents()
     char **cr_agents;
     char **av_agents;
 
-    av_agents = get_agents_by_last_keepalive(GA_ACTIVE, mond.delete_old_agents > 0 ? mond.delete_old_agents * 60 : DISCON_TIME);
+    av_agents = get_agents_by_last_keepalive(GA_ACTIVE, mond.delete_old_agents > 0 ? mond.delete_old_agents * 60 : Config.alert_agent_disconnection_time * 60);
```

Comment:
This computation of `*60` should be removed after changing the readding of settings.

Reply:
Also, remember that the `alert_agent_disconnection_time` is a time that should pass after `agents_disconnection_time` to generate an alert. So, the right computation to generate an alert here should be:

```
agents_disconnection_time + alert_agent_disconnection_time
```

Reply:
Thank you!
This has been fixed.